src/components/WorkOrder/Photos/PhotoViewList.js

```diff
@@ -75,7 +72,7 @@ const UpdateDescriptionForm = ({ description, onSubmit, fileGroupId }) => {
             name="description"
             label="Description"
             defaultValue={description ?? ''}
-            placeholder="Add a description"
+            placeholder="Add a description..."
```

Comment:
is it necessary to have the dots at the end?

Reply:
Just following the design